examples/mfa_hierarchy/config.txt

```diff
@@ -0,0 +1,12 @@
+--algo-params-file examples/mfa_hierarchy/in/algo.asciipb
+--hier-type MFA 
```

Comment:
Do we need to keep the mfa_hierarchy folder we created in examples? If so we should clean the files in it before you review them otherwise we just make it local.

Reply:
Yes I think it's nice to keep these files here!

Reply:
Great! Then we should adapt them to the conventions you are using for the other tutorial, and also cancel/add some comments.